_security/configuration/best-practices.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Best Practices
+parent: Configuration
+nav_order: 11
+---
+
+# Best practices for OpenSearch security
+
+Setting up secure practices in OpenSearch is crucial for protecting your data. Here are 10 best practices that offer clear steps on keeping your system safe.
+
+## 1. Use your own public key infrastructure to setup SSL/TLS for OpenSearch
+
+Although using your own public key infrastructure (PKI), such as [AWS Certificate Manager](https://docs.aws.amazon.com/crypto/latest/userguide/awspki-service-acm.html), requires more effort at the beginning, a custom PKI provides you with all the flexibility needed to set up SSL/TLS in the most secure and performant way.
+
+### Enable SSL/TLS for node and REST layer traffic
+
+The SSL/TLS is enabled by default for the transport layer, which is used by node to node communication. The SSL/TLS on the REST layer is disabled by default.
+
+The following setting is required to enable encryption at REST layer: `plugins.security.ssl.http.enabled: true`
+
+For additional configuration options, such as specifying certificate paths, keys and certificate-authority files you can refer the steps on [configuring TLS certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls/)
+
+### Replace all demo certificates with your own PKI
+The certificates generated when initializing OpenSearch cluster with `install_demo_configuration.sh` are not suitable for production. These should be replaced with your own certificates.
+
+You can generate custom certificates in a few different ways. One approach is by using OpenSSL, described in detail on [Generating self-signed certificates]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/), alternatively there are tools available online which can simplify the certificate creation process.
+
+## 2. Prefer client certificate authentication for API authentication
+
+Client certificate authentication offers a secure alternative to passwords and is more suitable for machine-to-machine interactions. It also ensures low performance overhead since the authentication occurs at the TLS level. Nearly all client software, such as curl, and client libraries support this authentication method.
```

Comment:
```suggestion
Client certificate authentication offers a secure alternative to password authentication and is more suitable for machine-to-machine interactions. It also ensures low performance overhead because the authentication occurs at the TLS level. Nearly all client software, such as curl, and client libraries support this authentication method.
```